packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -43,7 +43,7 @@ function InjectDropdownMenuItem({
 	}
 	return (
 		<li role="presentation" key={key} {...liProps}>
-			<Inject component={component} getComponent={getComponent} {...rest} />
+			<Inject component={component} getComponent={getComponent} onSelect={onSelect} {...rest} />
```

Comment:
`onSelect` is for menuItem, can you change the handler name as `onSelectItem` so it will be passed to YearPicker with `{...rest}`

Reply:
hi, change to be onSelectItem will have a side effect which make the actionDropDown keep open even after select a year in the list.